aptos-move/aptos-gas/src/aptos_framework.rs

```diff
@@ -119,5 +119,7 @@ crate::natives::define_gas_parameters_for_natives!(GasParameters, "aptos_framewo
     [.object.exists_at.base, { 7.. => "object.exists_at.base" }, 250 * MUL],
     // These are dummy value, they copied from storage gas in aptos-core/aptos-vm/src/aptos_vm_impl.rs
     [.object.exists_at.per_byte_loaded, { 7.. => "object.exists_at.per_byte_loaded" }, 1000],
-    [.object.exists_at.per_item_loaded, { 7.. => "object.exists_at.per_item_loaded" }, 8000]
+    [.object.exists_at.per_item_loaded, { 7.. => "object.exists_at.per_item_loaded" }, 8000],
+    [.string_utils.base, "string_utils.format.base", 300 * MUL],
```

Comment:
This would need to be 8.. => since gas schedule version 7 is already in testnet. cc @vgao1996

Reply:
Yes, please bump LATEST_GAS_FEATURE_VERSION and use 8 here.